pkg/clig/cmd/cmd.go

```diff
@@ -5,14 +5,14 @@ import (
 	"github.com/spf13/cobra"
 	"k8s.io/utils/exec"
 
-	"github.com/izumin5210/clig/pkg/cli"
+	"github.com/izumin5210/clig/pkg/clib"
 	"github.com/izumin5210/clig/pkg/clig"
 )
 
-func NewDefaultCligCommand(wd cli.Path, build cli.Build) *cobra.Command {
+func NewDefaultCligCommand(wd clib.Path, build clib.Build) *cobra.Command {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewDefaultCligCommand should have comment or be unexported